testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/ClientTest.java

```diff
@@ -248,7 +248,8 @@ private void assertPaginatedClients(int start, int end, List<ClientRepresentatio
             expected.add("ccx-" + (i < 10 ? "0" + i : i));
         }
         List<String> a = actual.stream().map(rep -> rep.getClientId()).collect(Collectors.toList());
-        assertThat(a, is(expected));
+        assertThat(a, Matchers.hasSize(expected.size()));
+        assertThat(a, Matchers.containsInAnyOrder(expected.toArray()));
```

Comment:
Should not check in any order as the clients should be sorted

Reply:
Thanks, there is no index on client_id column in database, so it has to added. And also it would be good to check all other entities which can be paginated.